compiler/src/iree/compiler/Codegen/Common/test/fold_reshape_into_interface_tensor.mlir

```diff
@@ -47,6 +47,54 @@ func.func @fold_expand_into_loads_dynamic() -> tensor<2x?x16x32xf32> {
 //  CHECK-SAME:       offsets = [0, 0, 0, 0], sizes = [2, %[[SHAPE]], 16, 32], strides = [1, 1, 1, 1]
 //  CHECK-SAME:       !iree_tensor_ext.dispatch.tensor<readonly:tensor<2x?x16x32xf32>>{%[[SHAPE]]}
 
+// -----
+
+#pipeline_layout = #hal.pipeline.layout<constants = 3, bindings = [
+    #hal.pipeline.binding<storage_buffer, "ReadOnly|Indirect">], flags = Indirect>
+func.func @fold_expand_into_loads_fully_dynamic() -> tensor<?x?xf32> {
+  %c0 = arith.constant 0 : index
+  %0 = hal.interface.constant.load layout(#pipeline_layout) ordinal(0) : index
+  %5 = hal.interface.constant.load layout(#pipeline_layout) ordinal(1) : index
+  %6 = hal.interface.constant.load layout(#pipeline_layout) ordinal(2) : index
+  %1 = hal.interface.binding.subspan layout(#pipeline_layout) binding(0) alignment(64) offset(%c0)
+      flags("ReadOnly|Indirect") : !iree_tensor_ext.dispatch.tensor<readonly:tensor<?xf32>>{%0}
+  %2 = iree_tensor_ext.dispatch.tensor.load %1, offsets = [0], sizes = [%0], strides = [1]
+      : !iree_tensor_ext.dispatch.tensor<readonly:tensor<?xf32>>{%0} -> tensor<?xf32>
+  %4 = tensor.expand_shape %2 [[0, 1]] output_shape [%5, %6] : tensor<?xf32> into tensor<?x?xf32>
+  return %4 : tensor<?x?xf32>
+}
```

Comment:
Nit: Could you canonicalize these tests for readibility:
```
%0 = ...
%1 = ...
%2 = ...
```
Instead of:
```
%0 = ...
%5 = ...
%6 = ...
%1 = ...
```
Same for the one below.